Straight_skeleton_2/examples/Straight_skeleton_2/print.h

```diff
@@ -31,10 +31,11 @@ void print_polygons ( std::vector< boost::shared_ptr< CGAL::Polygon_2<K> > > con
     print_polygon(**pi);
 }
 
-template<class K>
-void print_polygon_with_holes ( CGAL::Polygon_with_holes_2<K> const& polywh )
+template<class K, class C>
+void print_polygon_with_holes ( CGAL::Polygon_with_holes_2<K,C> const& polywh )
 {
-  typedef CGAL::Polygon_with_holes_2<K> PolygonWithHoles ;
+  typedef CGAL::Polygon_with_holes_2<K,C> PolygonWithHoles ;
+  typedef typename PolygonWithHoles::Polygon_2 Polygon_2;
```

Comment:
That new typedef seems unused. That will create warnings about unused local typedef.